### PR TITLE
[WebexAdapter] Fix Regexes for messages with html content

### PR DESCRIPTION
**Pull request by ceciliaavila**
## Proposed Changes
Fix Regexes for striping mentions when messages have HTML content.

### CODE REVIEW

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.Tests/WebexHelperTests.cs [hunk 1]
```diff
@@ -92,13 +92,13 @@ public void DecryptedMessageToActivityWithHtmlShouldReturnActivity()
...
             Assert.Equal(message.Id, activity.Id);
-            Assert.Equal(message.Html, activity.Text);
+            Assert.Equal(message.Text, activity.Text);
         }
 
         [Fact]
         public void DecryptedMessageToActivityWithHtmlShouldReturnActivityWhenMatch()
         {
-            var serializedPerson = "{\"id\":\"/ciscospark://us/PEOPLE/different_id\"}";
+            var serializedPerson = "{\"id\":\"ciscospark://us/PEOPLE/different_id\"}";
```
Comment (GasparAcevedoZainSouthworks) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.Tests/WebexHelperTests.cs]: I think that we must review this Regex. I have some doubts on the /People/different, mostly on how the regex compiler would interpret this

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs [hunk 2]
```diff
@@ -130,26 +127,27 @@ public static Activity DecryptedMessageToActivity(Message decryptedMessage, Pers
             {
                 // strip the mention & HTML from the message
                 var pattern = new Regex($"^(<p>)?<spark-mention .*?data-object-id=\"{identity.Id}\".*?>.*?</spark-mention>", RegexOptions.IgnoreCase | RegexOptions.Multiline);
-                if (!decryptedMessage.Html.Equals(pattern))
+                if (!pattern.IsMatch(decryptedMessage.Html))
                 {
                     // this should look like ciscospark://us/PEOPLE/<id string>
-                    var match = Regex.Match(identity.Id, "/ciscospark://.*/(.*)", RegexOptions.IgnoreCase | RegexOptions.Multiline);
+                    var match = Regex.Match(identity.Id, "ciscospark://.*/(.*)", RegexOptions.IgnoreCase | RegexOptions.Multiline);
```
Comment (GasparAcevedoZainSouthworks) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs]: Same on the regex thing.

Comment (ceciliaavila) [reply libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs]: This is the one that still needs review. But I wanted to send to MS what we have so far so Gabo can see the progress.

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.TestBot/Bots/EchoBot.cs [hunk 3]
```diff
@@ -37,7 +37,8 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
             }
             else
             {
-                var activityCard = MessageFactory.Attachment(CreateAdaptiveCardAttachment(Directory.GetCurrentDirectory() + @"\Resources\adaptive_card.json"));
+                // var activityCard = MessageFactory.Attachment(CreateAdaptiveCardAttachment(Directory.GetCurrentDirectory() + @"\Resources\adaptive_card.json"));
```
[the remark this answers is not in the document]
Comment (ceciliaavila) [reply tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.TestBot/Bots/EchoBot.cs]: Done, thanks!

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs (hunk 2 above)
Comment (GasparAcevedoZainSouthworks) [reply libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexHelper.cs]: OK, send these changes, but don't forget to review this with me later.

**Pull request merged**